cmd/lncli/commands.go

```diff
@@ -123,7 +123,7 @@ var newAddressCommand = cli.Command{
 }
 
 func newAddress(clictx *cli.Context) error {
-	ctxt := getContext()
+	ctx := getContext()
```

Comment:
this should be squashed with the first commit.

Reply:
done